upstream/CVE-2019-11249.json

```
@@ -0,0 +1,89 @@
+{
+  "id": "CVE-2019-11249",
+  "modified": "2019-08-05T12:44:23Z",
+  "published": "2019-08-05T12:44:23Z",
+  "summary": "Incomplete fixes for CVE-2019-1002101 and CVE-2019-11246, kubectl cp potential directory traversal",
+  "details": "The kubectl cp command allows copying files between containers and the user machine. To copy files from a container, Kubernetes runs tar inside the container to create a tar archive, copies it over the network, and kubectl unpacks it on the user’s machine. If the tar binary in the container is malicious, it could run any code and output unexpected, malicious results. An attacker could use this to write files to any path on the user’s machine when kubectl cp is called, limited only by the system permissions of the local user. Kubernetes affected versions include versions prior to 1.13.9, versions prior to 1.14.5, versions prior to 1.15.2, and versions 1.1, 1.2, 1.4, 1.4, 1.5, 1.6, 1.7, 1.8, 1.9, 1.10, 1.11, 1.12.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/kubectl"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.0/AV:N/AC:H/PR:L/UI:R/S:U/C:N/I:H/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.1.0"
```

I think it's wrong. It's written as below in the GitHub issue.

>Kubernetes 1.0.x-1.12.x

```suggestion
              "introduced": "1.0.0"
```

---

fixed in https://github.com/aquasecurity/vuln-list-k8s/pull/70/commits/d76dd0614c42d463d506381ab6a9f5779bb7c845